features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/model/event/TimelineItemEncryptedContentProvider.kt

```diff
@@ -36,7 +36,7 @@ open class TimelineItemEncryptedContentProvider : PreviewParameterProvider<Timel
             aTimelineItemEncryptedContent(
                 data = UnableToDecryptContent.Data.MegolmV1AesSha2(
                     sessionId = "sessionId",
-                    utdCause = UtdCause.HistoricalMessage,
+                    utdCause = UtdCause.HistoricalMessageAndBackupIsDisabled,
```

Comment:
Add the case `HistoricalMessageAndDeviceIsUnverified` here? Can be done later be we may forget :)